sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -547,6 +580,19 @@ private static Object toBeamValue(FieldType fieldType, Object jsonBQValue) {
    */
   public static Object convertAvroFormat(
       FieldType beamFieldType, Object avroValue, BigQueryUtils.ConversionOptions options) {
+    return convertAvroFormat(beamFieldType, null, avroValue, options);
+  }
+
+  /**
+   * Tries to convert an Avro decoded value to a Beam field value based on the target type of the
+   * Beam field, as well as the source avro field logical type.
+   */
+  public static Object convertAvroFormat(
+      FieldType beamFieldType,
+      org.apache.avro.Schema avroFieldSchema,
```

Comment:
This field should be annotated with `@Nullable`. @kennknowles shouldn't checkstyle complain about this?